elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/ElasticSearchAdapterTest.java

```diff
@@ -389,7 +420,8 @@ public void limit2() {
                 "size:5"));
   }
 
-  @Test public void testFilterSort() {
+  //@Test
```

Comment:
Why remote this (`testFilterSort`) test ?

Reply:
Done, reverted this change.